packages/suite/src/actions/suite/guideActions.ts

```diff
@@ -0,0 +1,62 @@
+import qs from 'qs';
+import { GUIDE } from './constants';
+import type { ActiveView, FeedbackPayload, FeedbackType } from '@suite-types/guide';
+import { Dispatch } from '@suite-types';
+import { addToast } from '@suite-actions/notificationActions';
+import { getLocationHostname } from '@suite-utils/env';
+
+export type GuideAction =
+    | { type: typeof GUIDE.OPEN }
+    | { type: typeof GUIDE.CLOSE }
+    | { type: typeof GUIDE.SET_VIEW; payload: ActiveView }
+    | { type: typeof GUIDE.OPEN_ARTICLE; payload: string };
+
+export const open = () => ({
+    type: GUIDE.OPEN,
+});
+
+export const close = () => ({
+    type: GUIDE.CLOSE,
+});
+
+export const setView = (payload: ActiveView) => ({
+    type: GUIDE.SET_VIEW,
+    payload,
+});
+
+export const openArticle = (payload: string) => (dispatch: Dispatch) => {
+    dispatch(setView('GUIDE_ARTICLE'));
+    dispatch({
+        type: GUIDE.OPEN_ARTICLE,
+        payload,
+    });
+};
+
+const getUrl = (feedbackType: FeedbackType) => {
+    const typeUri = feedbackType === 'BUG' ? 'bugs' : 'feedback';
+    const base = `https://data.trezor.io/suite/${typeUri}`;
+    const hostname = getLocationHostname();
+    switch (hostname) {
+        case 'beta-wallet.trezor.io':
+        case 'suite.trezor.io':
+            return `${base}/prod.log`;
+        case 'localhost':
+        case 'staging-suite.trezor.io':
```

Comment:
minor: I think it's better to avoid connecting other cases with the default one, it could lead to confusion because it doesn't do anything.
```suggestion
```

And another topic – it would be nice to put those urls in constants and reuse it to prevent typos, but definitely not as part of this PR.

Reply:
I'll clean it up

Reply:
Aligned with Analytics 1.9 as @tomasklim recommended.